Pod/Classes/Banner.swift

```diff
@@ -202,8 +208,8 @@ public class Banner: UIView {
     }
     
     private func addGestureRecognizers() {
-        addGestureRecognizer(UITapGestureRecognizer(target: self, action: "didTap:"))
-        let swipe = UISwipeGestureRecognizer(target: self, action: "didSwipe:")
+      addGestureRecognizer(UITapGestureRecognizer(target: self, action:  #selector(Banner.didTap(_:))))
```

Comment:
Nice catch! This is Xcode's converter work, will fix this and the other issues in a minute